helm/templates/alertconfig.yaml

```diff
@@ -5,10 +5,8 @@ metadata:
   name: {{ .Chart.Name }}-alert-templates
   namespace: {{ .Release.Namespace }}
   labels:
-    app.kubernetes.io/name: {{ include "helm.name" . }}
-    helm.sh/chart: {{ include "helm.chart" . }}
-    app.kubernetes.io/instance: {{ .Chart.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: core
```

Comment:
imo it might be a good idea to add some form of `connaisseur` to either label name or value to disambiguate it from other such labels

Reply:
I mean, if I do `kubectl delete all -l app.kubernetes.io/component=core` I might very well delete something else that also has poor naming, which is both on them and us. Running `kubectl delete all -l app.kubernetes.io/component=connaisseur-core`, if I delete something else then it's totally on them, not on us

Reply:
hm...gotcha...think I would still question your abilities, but might be willing to agree and will adjust